test/com/google/javascript/jscomp/parsing/TypeSyntaxTest.java

```diff
@@ -111,6 +111,16 @@ public void testTypeInDocAndSyntax() {
     parse("var /** string */ foo: string = 'hello';");
   }
 
+  public void testTypedGetterSetterDeclaration() {
+    Node n = parse("var x = {get a(): number {\n}};", LanguageMode.ECMASCRIPT6_TYPED);
+    assertDeclaredType("number type", numberType(),
+        n.getFirstChild().getFirstChild().getFirstChild().getFirstChild().getFirstChild());
+    n = parse("var x = {set a(v: number) {\n}};", LanguageMode.ECMASCRIPT6_TYPED);
+    assertDeclaredType("number type", numberType(),
+        n.getFirstChild().getFirstChild().getFirstChild().getFirstChild().getFirstChild().getChildAtIndex(1)
```

Comment:
nit: I think there were some test utilities that allowed you to search a tree for a node of a particular type - that might be more readable than this tree navigation style.